package.json

```diff
@@ -12,24 +12,21 @@
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
   "scripts": {
-    "clean": "lerna run clean",
-    "precompile": "tsc --version && npm run version:update",
-    "version:update": "lerna run version:update",
-    "compile": "lerna run compile",
-    "prewatch": "npm run precompile",
-    "test": "lerna run test",
-    "test:ci:changed": "lerna run test --since origin/main",
-    "test:browser": "lerna run test:browser --concurrency 1",
-    "test-all-versions": "npm run --if-present --workspaces test-all-versions",
-    "bump": "lerna publish",
```

Comment:
note for reviewer: I think this bump script is not used since release-please is handling the versioning

Reply:
Yep that seems correct. I'd be surprised if this was used by anything. 
Thanks for removing it :+1: